spring-cloud-dataflow-server-local-autoconfig/src/main/java/org/springframework/cloud/dataflow/autoconfigure/local/LocalDataFlowServerAutoConfiguration.java

```diff
@@ -59,4 +63,29 @@ public LRUCleaningResourceLoaderBeanPostProcessor lruCleaningResourceLoaderInsta
 		float fRatio = localDeployerProperties.getFreeDiskSpacePercentage() / 100F;
 		return new LRUCleaningResourceLoaderBeanPostProcessor(fRatio, repositoryCache);
 	}
+
+	@Bean
```

Comment:
What is the significance of this no-op implementation in `local` Data Flow server?

Reply:
Good question, we need to have an implementation for the local deployer. Else the application will fail to compile (until we provide the real one).